src/patternfly/demos/LoginPage/examples/login-page-simple-example.hbs

```diff
@@ -12,20 +12,97 @@
     {{#> login-main}}
       {{#> login-box}}
         {{#> login-box-header}}
-          {{#> title titleType="h1" title--modifier="pf-m-3xl"}}
-            Login box title
+          {{#> title titleType="h1" title--modifier="pf-m-3xl pf-u-mb-md"}}
+            Sign in to your account
           {{/title}}
+          {{#> display display--type="inline-flex" display--modifier="pf-u-align-items-center"}}
+            {{#> dropdown id="dropdown-example-collapsed" pf-c-dropdown--HasToggleIcon="true"}}
+              English
+            {{/dropdown}}
```

Comment:
I know the Select variation of the Dropdown component isn't merged yet, but we'll want to use that variation here when it's ready. 

Also, the toggle arrow is missing in the example. 😕

Reply:
The toggle arrow is now back (a small syntax issue made it disappear).

Is this a situation where I should create a new issue and assign it to myself to remember to add the Select variation once it's merged? Or is #750 close enough to merging that I should just sit tight with this PR?